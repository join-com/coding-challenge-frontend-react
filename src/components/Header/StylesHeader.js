import styled from "styled-components";
import { colors, defaultIndent } from "../../constants/styles";

export const StyledHeader = styled.div`
  background-color: ${colors.primary};
  padding: ${defaultIndent * 3}rem 0;
`;
