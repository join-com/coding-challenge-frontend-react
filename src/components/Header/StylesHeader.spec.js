import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import { StyledHeader } from "./StylesHeader";

describe("StyledHeader component", () => {
  it("should match snapshot", () => {
    expect(renderer.create(<StyledHeader />).toJSON()).toMatchSnapshot();
  });
});
