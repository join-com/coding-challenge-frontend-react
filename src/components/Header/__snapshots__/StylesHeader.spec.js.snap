// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledHeader component should match snapshot 1`] = `
.c0 {
  background-color: #90D3DC;
  padding: 1.5rem 0;
}

<div
  className="c0"
/>
`;
